src/sentry/testutils/factories.py

```diff
@@ -581,6 +582,13 @@ def create_release(
 
         return release
 
+    def create_group_release(project: Project, group: Group, release: Release):
```

Comment:
IIRC you need to put a return type to a function to be fully typed (I may be misremembering).

Either way this file is not being tested for typing:
https://github.com/getsentry/sentry/blob/c2158c3291a59f12c1a3779d9161072f0aaf03ad/pyproject.toml#L587

Reply:
I'll add in the type just to fix it, not too worried about having this whole file typed